Adding documentation for curl requests in git polling (#2314)
harness/developer-hub: docs/platform/8_Pipelines/w_pipeline-steps-reference/triggers-reference.md

![Getting a webhook ID from a GitHub webhook page URL.](./static/752891ea2d0d9bcee2511ad039994271c20f002eb525570b5bc8038915b85da1.png)

:::note

> Please note that we make an api call using github retrofit rest client to fetch the webhook events for this webhook and in case even that fails then we make a simple curl request to do the same.

Reviewer: ```suggestion
In case the API call using the GitHub retrofit REST client to fetch the webhook events fail, Harness makes a simple cURL request to do the same.
```
